src/pages/ReportParticipantsPage.tsx

```diff
@@ -50,7 +50,7 @@ const getAllParticipants = (
                 !!userPersonalDetail?.login && !CONST.RESTRICTED_ACCOUNT_IDS.includes(accountID) ? LocalePhoneNumber.formatPhoneNumber(userPersonalDetail.login) : translate('common.hidden');
             const displayName = PersonalDetailsUtils.getDisplayNameOrDefault(userPersonalDetail);
 
-            const pendingChatMember = report?.pendingChatMembers?.find((member) => member.accountID === accountID.toString());
+            const pendingChatMember = report?.pendingChatMembers?.findLast((member) => member.accountID === accountID.toString());
             return {
```

Comment:
So, I applied the fix to the report participants page too, but when I tested it, I found out that removing a WS member removes the user completely from the report participants, so we can't see it getting strikethrough. (see video below).

https://github.com/Expensify/App/assets/50919443/7ac42a7e-fe0f-4600-a645-c532f9d91399

This is the same issue as https://github.com/Expensify/App/issues/31764, just on a different page. (they fix for room members, but not for announce report participants)

We can fix this with the same solution by removing this optimistic code that removes the user from the list. Do we want to do it here? Or should https://github.com/Expensify/App/issues/31764 handle it as part of the issue?
https://github.com/Expensify/App/blob/5a8cb7fb94eeb16cfad5407dd9495be1f0a921b9/src/libs/actions/Policy.ts#L783
https://github.com/Expensify/App/blob/5a8cb7fb94eeb16cfad5407dd9495be1f0a921b9/src/libs/actions/Policy.ts#L786-L792

@francoisl @mollfpr

Reply:
Yeah, would be great to do it here in this PR if you can. Thanks!

Reply:
Fixed.
<img width="1102" alt="image" src="https://github.com/Expensify/App/assets/50919443/f4ceb2ae-b825-42dd-9b10-183fd6067eee">

I also need to update OptionRow because OfflineWithFeedback strikethrough can't work if the child is a function. So, 2 bug is fixed.